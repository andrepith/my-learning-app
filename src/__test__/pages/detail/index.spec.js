import ShallowRenderer from 'react-test-renderer/shallow';
import Detail from '../../../pages/detail';

it('Renders detail to match snapshot', () => {
  const renderer = new ShallowRenderer();
  const tree = renderer.render(<Detail />);
  expect(tree).toMatchSnapshot();
});
