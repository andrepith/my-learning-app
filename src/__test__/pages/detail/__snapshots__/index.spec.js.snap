// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders detail to match snapshot 1`] = `
<div
  className="detail"
>
  <h1>
    Class Detail
  </h1>
  <form
    className="register-form"
    onSubmit={[Function]}
  >
    <h3>
      Register User
    </h3>
    <div
      className="form-data"
    >
      <input
        name="attendeeFullName"
        onChange={[Function]}
        placeholder="Name"
        required={true}
        type="text"
        value=""
      />
    </div>
    <div
      className="form-data"
    >
      <input
        name="attendeeEmail"
        onChange={[Function]}
        placeholder="Email"
        required={true}
        type="text"
        value=""
      />
    </div>
    <input
      className="btn btn-primary dashboard-button"
      type="submit"
      value="Enroll"
    />
  </form>
</div>
`;
